Source/NSPopover.m

```diff
@@ -232,23 +232,27 @@ - (void) close
 {
   [_realPanel close];
   [_realPanel setDelegate:nil];
+  [_realPanel orderOut: nil];
```

Comment:
Why is the `orderOut:` needed here and in the next method? The close Methods should already take care of that.

Reply:
This should now be addressed.